index.js

```diff
@@ -27,7 +27,8 @@ module.exports = function (url, opts, cb) {
     opts = objectAssign({}, opts, {
         url: protocolify(url),
         width: viewportSize[0] || 1024,
-        height: viewportSize[1] || 768
+        height: viewportSize[1] || 768,
+        delay: opts.delay || 0
```

Comment:
Hmm. `delay: 0` (delay: 1 now) breaks my tests whilst `delay: opts.delay || 1` appears to work. Am I doing something silly? :)